.circleci/config.yml

```diff
@@ -136,6 +190,71 @@ jobs:
             cd tests/
             codecov
 
+  test-linux-aarch64:
+    parameters:
+      manylinux-tag:
+        type: string
+      manylinux-arch:
+        type: string
+      python-version:
+        type: string
+      numpy-version:
+        type: string
+
+    machine:
+      image: ubuntu-2004:202101-01
+
+    resource_class: arm.medium
+
+    steps:
+      - checkout
+
+      - attach_workspace:
+          at: dist
+
+      # Make sure that we're using the built dimod and not one from pypi
+      - run:
+          name: install
+          command: |
+            npv='<< parameters.numpy-version >>'
+            pyv='<< parameters.python-version >>'
+            docker run -e npv=$npv -e pyv=$pyv --rm -v $PWD:/ws:rw --workdir=/ws \
+            quay.io/pypa/<< parameters.manylinux-tag >>_<< parameters.manylinux-arch >> \
+            bash -exc '
+              /opt/python/${pyv}/bin/python -m venv env
+              . env/bin/activate
+              pip install -U pip wheel
+              pip install Cython>=0.29.21
+              pip install dimod==0.10.4
+              pip install -r tests/requirements.txt
+              pip install numpy${npv} --upgrade --only-binary=numpy
+              pip install dwave-preprocessing --no-build-isolation
+              pip install -U pip pyparsing
+              pip install dimod --no-index -f dist/ --no-deps --force-reinstall
+            '
+
+      - run:
+          name: run tests
+          command: |
+            docker run --rm -v $PWD:/ws:rw --workdir=/ws \
+            quay.io/pypa/<< parameters.manylinux-tag >>_<< parameters.manylinux-arch >> \
+            bash -exc '
+              . env/bin/activate
+              cd tests/
+              coverage run --source=dimod --append -m unittest
+            '
+
+      - run:
```

Comment:
I think it's OK to skip the coverage report for these jobs actually, keep it a bit simpler.